src/API/Authentication.php

```diff
@@ -333,19 +340,26 @@ public function login($username, $password, $realm, array $options = [])
      *
      * @param string $username
      * @param string $password
-     * @param array $options {
-     *
-     *      @var string $scope    Optional.
-     *      @var string $audience Optional.
-     *}
+     * @param string $scope
+     * @param string $audience
      *
      * @return array
      */
-    public function loginWithDefaultDirectory($username, $password, array $options = [])
+    public function loginWithDefaultDirectory($username, $password, $scope = null, $audience = null)
```

Comment:
> should we default here to scope ="openid"

no, it does not make a lot of sense in this endpoint 

>  if we should retain options as a just in case

I dont think so, we can add later if we need